protos/repository.proto

```diff
@@ -50,6 +50,7 @@ message BentoServiceMetadata {
     int32 mb_max_latency = 7;
     int32 mb_max_batch_size = 8;
     bool batch = 9;
+    string route = 10;
```

Comment:
I think this is part of the #1334 ?  I guess we are holding this after that PR merge in?